brix-core/src/main/java/org/brixcms/jcr/RepositoryUtil.java

```diff
@@ -29,6 +29,33 @@
 public class RepositoryUtil {
     private static final Logger logger = LoggerFactory.getLogger(Brix.class);
 
+    /**
+     * register a brix basic node as a unstructured node type, reason is: accoring to JCR spec 2,
+     * a mixin may only have other mixins as supertypes
+     *
+     * @param workspace
+     */
+    public static void registerBrixUnstructuredMixin(Workspace workspace) {
+        String cnd = "[brix:unstructured] mixin" +
+                "  - * (undefined) multiple " +
+                "  - * (undefined) " +
+                "  + * (nt:base) sns version ";
+
+        NodeTypeManager manager = null;
+        try {
+            manager = workspace.getNodeTypeManager();
+
+            if (manager.hasNodeType("brix:unstructured") == false) {
+                CndImporter.registerNodeTypes(new StringReader(cnd), workspace.getSession());
+            }
+
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
By try/catching and just logging it you do not fail the hard way.
I'd expect to re-throw the exception (or not catch it at all) to fail.